server/views/templates/explore-landing.njk

```diff
@@ -1,6 +1,8 @@
 {% extends "pages/explore-landing.njk" %}
 
 {% block body %}
+  {% component 'page-description', { title: 'Explore landing', modifiers: ['hidden'] } %}
```

Comment:
I was thinking more as a descriptor for the page, Explore landing doesn't give us much.
SEO tends to be, as it's so damn smart, make good user experiences.